canine/localization/delocalization.py

```diff
@@ -58,13 +72,17 @@ def main(output_dir, jobId, patterns, copy):
                 # compute checksum
                 if name not in {'stdout', 'stderr'}:
                     try:
-                        subprocess.check_call(
-                          "sha1sum {target} | awk '{{ print $1 }}' > {output}".format(
-                            target = target,
-                            output = os.path.join(jobdir, name, os.path.dirname(os.path.relpath(target)), "." + os.path.basename(target) + ".sha1")
+                        crc32 = get_gsfile_crc32c(target)
+                        with open(
+                          os.path.join(
+                            jobdir,
+                            name,
+                            os.path.dirname(os.path.relpath(target)),
+                            "." + os.path.basename(target) + ".crc32c"
```

Comment:
This isn't going to work; calling `gsutil hash ...` for every single file individually is extremely slow. We need to do it in a single pass.

Reply:
Fixed in 
d7ea6db